acra-translator/docker-compose.acra-translator.yml

```diff
@@ -0,0 +1,145 @@
+version: "3"
+
+volumes:
+  acra-translator:
+  mongodb-data:
+    driver: local
+
+services:
+  # Creating keys:
+  # - ./.acrakeys/acra-server/${ACRA_CLIENT_ID}_storage
+  # - ./.acrakeys/acra-writer/${ACRA_CLIENT_ID}_storage.pub
+  acra-keymaker_client:
+    image: "cossacklabs/acra-keymaker:${ACRA_DOCKER_IMAGE_TAG:-0.95.0}"
+    network_mode: "none"
+    environment:
+      ACRA_MASTER_KEY: ${ACRA_SERVER_MASTER_KEY:-N0MyZEVCRGY1cHA4ODRSVHp3c3BNeGtDekxaV3BhTHI=}
+    volumes:
+      - acra-translator:/keys
+      - ../_common/ssl/acra-client/:/ssl:ro
+    command: >-
+      --client_id=""
+      --tls_cert=/ssl/acra-client.crt
+      --keys_output_dir=/keys/acra-translator
+      --keys_public_output_dir=/keys/acra-translator
+      --keystore=v1
+
+  mongo:
+    image: mongo
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: ${MONGO_ROOT_USER:-root}
+      MONGO_INITDB_ROOT_PASSWORD: ${MONGO_ROOT_PASSWORD:-password}
+    volumes:
+      - mongodb-data:/data/db
+    healthcheck:
+      test: echo 'db.runCommand("ping").ok' | mongo mongo:27017/test --quiet
+      interval: 10s
+      timeout: 10s
+      retries: 5
+      start_period: 40s
+    ports:
+      - "27017:27017"
+    networks:
+      - go-mongo
+      - world
+      - webui-mongo
+
+  mongo-express:
+    image: mongo-express
+    environment:
+      - ME_CONFIG_MONGODB_SERVER=mongo
+      - ME_CONFIG_MONGODB_PORT=27017
+      - ME_CONFIG_MONGODB_ENABLE_ADMIN=false
+      - ME_CONFIG_MONGODB_AUTH_DATABASE=admin
+      - ME_CONFIG_MONGODB_AUTH_USERNAME=${MONGO_ROOT_USER:-root}
+      - ME_CONFIG_MONGODB_AUTH_PASSWORD=${MONGO_ROOT_PASSWORD:-password}
+      - ME_CONFIG_BASICAUTH_USERNAME=${MONGOEXPRESS_LOGIN:-test}
+      - ME_CONFIG_BASICAUTH_PASSWORD=${MONGOEXPRESS_PASSWORD:-test}
+    depends_on:
+      - mongo
+    ports:
+      - "8081:8081"
+    networks:
+      - world
```

Comment:
why it works without `webui-mongo` network ?)
as I understand, it should render some UI but have problems with connectivity to mongo server